src/Components/RoutingHelper.js

```diff
@@ -53,8 +53,8 @@ export const RoutingHelper = () => {
     );
   } else if (noHeaderMatch) {
     return <Outlet />;
-  } else if (!user && headerMatch) {
-    return <Login />;
+  } else if (!loading && !user && headerMatch) {
+    logInAnon();
```

Comment:
Nice!  Love how simple this is.  Might also be a good idea to return a BreathingLogo here so that the user sees something while the login happens.  

I know we are pretty confident we won't re-render a bunch here, since this is a super-top-level component, but just to make sure we don't hammer logInAnon on re-renders, it might be nice to wrap the `logInAnon` in a useEffect higher up (can't put a hook inside an if/else), and here you could just return a BreathingLogo.

Maybe:
```js
useEffect(() => {
  if (!loading && !user && headerMatch) {
    logInAnon();
  }
}, [loading, user, headerMatch]);
```

And then:
```js
if (loading && !user) {
  return <BreathingLogo type={"fullPage"} />;
} else if (user && headerMatch) {
  return (
    <>
      <Header />
      <Outlet />
    </>
  );
} else if (noHeaderMatch) {
  return <Outlet />;
} else if (!user && headerMatch) {
  return <BreathingLogo type={"fullPage"} />;
} else {
  return <Outlet />;
}
```